debian/raspberrypi-sys-mods.postinst

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh -e
 
-block_ms_repo () {
+remove_ms_stubs () {
   eval "$(apt-config shell APT_SOURCE_PARTS Dir::Etc::sourceparts/d)"
   CODE_SOURCE_PART="${APT_SOURCE_PARTS}vscode.list"
   eval "$(apt-config shell APT_TRUSTED_PARTS Dir::Etc::trustedparts/d)"
   CODE_TRUSTED_PART="${APT_TRUSTED_PARTS}microsoft.gpg"
-  [ ! -f "$CODE_SOURCE_PART" ] || echo "### Disabled by raspberrypi-sys-mods ###" > "$CODE_SOURCE_PART" || :
-  [ ! -f "$CODE_TRUSTED_PART" ] || > "$CODE_TRUSTED_PART" || :
+  grep -qsv '^[[:space:]]*#' "$CODE_SOURCE_PART" || rm -f "$CODE_SOURCE_PART"
+  grep -qsv '^[[:space:]]*#' "$CODE_TRUSTED_PART" || rm -f "$CODE_TRUSTED_PART"
```

Comment:
@XECDesign 
This won't work as this file is empty 😉.

Reply:
That's the idea. If the file is empty, remove it. Which part won't work?

Reply:
Sorry, I was editing back and forth 😄. I mean the purpose of the patch was to block the repository, as of the massive negative feedback about having a Microsoft repo and trusted key added without interactive permission, and the good solution to ship VSCode via `archive.raspberrypi.org`. With this change, the repository will be left in place when upgrading from pre-20210310, since the files then do contain data. So to keep the important original purpose of the patch, files must be removed either unconditionally or the empty files removed as a dedicated patch.

Reply:
The idea is only to remove cleared files, since anything else might have been added by the user intentionally (maybe they use the repo to install older versions which we don't host, idk).

Pre-20210310 updates are not relevant since this change only applies to bullseye. Mostly because we don't support upgrading from buster, but also because sensible upgrade steps would involve making sure the buster install is up to date first (https://www.debian.org/releases/stable/mips64el/release-notes/ch-upgrading.en.html#upgrade-to-latest-point-release).

Reply:
> Pre-20210310 updates are not relevant since this change only applies to bullseye.

Ah okay, then I wonder why the patch was left in the Bullseye maintainer script at all 😄. Okay then it should be fine as you changed it.

Reply:
Because I didn't know that the fix to vscode has made it into all their debs. Last time I checked, it hadn't, but it was quite a while ago.